### PR TITLE
5551 nursing internal add col width proportional values

### PR DESCRIPTION
**Pull request by jbwilson8**
Issue: [5551](https://github.com/openstax/cnx-recipes/issues/5551)

### CODE REVIEW

File: styles/designs/carnival/parts/_columns-components.scss [hunk 1]
```diff
@@ -66,6 +66,23 @@ $Columns--ThreeColumnsFourRatioTwo: (
     )
 );
 
+$Columns--ThreeColumnsFiveRatioOne: (
+    _name: "FourColumnsFourRatioOne",
```
Comment (wilkus27) [inline styles/designs/carnival/parts/_columns-components.scss:70]: Name key doesn't match component name

File: styles/designs/carnival/parts/_columns-components.scss [hunk 2]
```diff
@@ -90,6 +107,22 @@ $Columns--ThreeColumnsSixRatioThree: (
     )
 );
 
+$Columns--ThreeColumnsSeventeenRatioOne: (
+    _name: "ThreeColumnsSixRatioThree",
```
Comment (wilkus27) [inline styles/designs/carnival/parts/_columns-components.scss:111]: Name key doesn't match component name

File: styles/designs/carnival/parts/_columns-components.scss [hunk 3]
```diff
@@ -90,6 +107,22 @@ $Columns--ThreeColumnsSixRatioThree: (
...
+    _properties: (
+        width: column-width(3,17,1),
+    )
+);
+
+$Columns--ThreeColumnsSeventeenRatioEight: (
+    _name: "ThreeColumnsSeventeenRatioEight",
+    _subselector: " [data-columns='3'][data-ratio='1,8,8'] col[data-width='8*']",
+    _properties: (
+        width: column-width(3,17,1),
```
Comment (wilkus27) [inline styles/designs/carnival/parts/_columns-components.scss:122]: It should be `(3,17,8)`

File: styles/designs/carnival/parts/_columns-shapes.scss [hunk 4]
```diff
@@ -11,13 +11,17 @@
                 $Columns--TwoColumnsFiveRatioThree,
                 $Columns--ThreeColumnsThreeRatioOne,
                 $Columns--ThreeColumnsFourRatioOne,
-                $Columns--ThreeColumnsFourRatioTwo,
```
Comment (wilkus27) [inline styles/designs/carnival/parts/_columns-shapes.scss:14]: This one is accidentally removed

File: styles/designs/carnival/parts/_columns-shapes.scss [hunk 5]
```diff
@@ -11,13 +11,17 @@
...
                 $Columns--ThreeColumnsSixRatioOne,
                 $Columns--ThreeColumnsSixRatioTwo,
                 $Columns--ThreeColumnsSixRatioThree,
+                $Columns--ThreeColumnsSeventeenRatioOne,
+                $Columns--ThreeColumnsSeventeenRatioEight,
                 $Columns--FourColumnsFourRatioOne,
+                $Columns--FourColumnsFiveRatioOne,
+                $Columns--FourColumnsFiveRatioTwo,
                 $Columns--FourColumnsSixRatioOne,
-                $Columns--FourColumnsSixRatioThree,
```
Comment (wilkus27) [inline styles/designs/carnival/parts/_columns-shapes.scss:20]: This one is accidentally removed

File: styles/designs/webview/parts/_columns-components.scss [hunk 6]
```diff
@@ -58,6 +58,39 @@ $Columns--ThreeColumnsFourRatioTwo: (
...
+    _properties: (
+        width: column-width(3,17,1),
+    )
+);
+
+$Columns--ThreeColumnsSeventeenRatioEight: (
+    _name: "ThreeColumnsSeventeenRatioEight",
+    _subselector: " [data-columns='3'][data-ratio='1,8,8'] col[data-width='1*']",
+    _properties: (
+        width: column-width(3,17,1),
```
Comment (wilkus27) [inline styles/designs/webview/parts/_columns-components.scss:90]: It should be (3,17,8)

**Review by kswanson33 — APPROVED** [review]

File: styles/designs/webview/parts/_columns-components.scss (hunk 6 above)
Comment (wilkus27) [inline styles/designs/webview/parts/_columns-components.scss:90]: Oh I think this one was fixed in a wrong way. It should be for 8.
```
_subselector: " [data-columns='3'][data-ratio='1,8,8'] col[data-width='8*']",
    _properties: (
        width: column-width(3,17,8),
```

**Review by wilkus27 — APPROVED** [review]
I small thing. Beside that everything looks good :+1:

**Pull request merged**